src/pipeline_edsl/to_workflow.ml

```diff
@@ -231,14 +231,21 @@ end
 open Biokepi_run_environment
 
 module type Compiler_configuration = sig
-  val work_dir: string
+  val work_dir : string
+  (** Directory where all work product done by the TTFI end up. *)
+  val results_dir : string
+  (** Directory where `save` files will end up. *)
+
   val machine : Machine.t
+  (** Biokepi machine used by the TTFI workflow. *)
+
   val map_reduce_gatk_indel_realigner : bool
+  (** Whether or not to scatter-gather the indel-realigner results. *)
 
+  val input_files: [ `Copy | `Link | `Do_nothing ]
   (** What to do with input files: copy or link them to the work directory, or
       do nothing. Doing nothing means letting some tools like ["samtools sort"]
       write in the input-file's directory. *)
-  val input_files: [ `Copy | `Link | `Do_nothing ]
 end
 
 
```

Comment:
I think `results_dir` should be a `string option` so that we can put `None` (and do nothing) in the defaults below:
:arrow_down: